components/engine/Cargo.toml

```diff
@@ -32,8 +33,9 @@ default-features = false
 features = ["nightly"]
 
 [dependencies.engine_rocksdb]
-git = "https://github.com/pingcap/rust-rocksdb.git"
+git = "https://github.com/yiwu-arbug/rust-rocksdb.git"
```

Comment:
Should change back to pingcap?

Reply:
Need an approval on https://github.com/pingcap/rust-rocksdb/pull/320

Reply:
updated